modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue

```diff
@@ -163,6 +167,11 @@ export default {
   },
   methods: {
     computeActiveOperations () {
+      if (this.isRelationship) {
```

Comment:
I think we should clear the batch operations on the manager level rather than put the work in the toolbar.